projects/plugins/jetpack/extensions/blocks/subscriptions/edit.js

```diff
@@ -72,6 +72,31 @@ export function SubscriptionEdit( props ) {
 	} = props;
 	const { isLoadingModules, isChangingStatus, isModuleActive, changeStatus } =
 		useModuleStatus( name );
+
+	const { subscriberCount, subscriberCountString } = useSelect( select => {
```

Comment:
> I see that we're importing it above, but it doesn't seem to be working. Do we want to use withSelect here? I see that it's already imported although I'm not sure of the difference between the two currently. I tried withSelect locally and it seemed to work as I would expect useSelect to work here.

There might be a conflict using both approaches at the same time... But [it is not in the doc](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data/#useregistry)...

Reply:
@chrismccluskey The issue ended up being `useSelect` was imported from the wrong package (`WP/elements` instead of `WP/data`).

But in the meantime I tried the `withSelect` approach that could not be used because of `isModuleActive` being retrieved from `useModuleStatus`. But you can have a look at the implementation on d0dd537da4